content/posts/2023/pdf-root-files-preview.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Introducing PDF and ROOT File Previews in REANA Web"
+date: 2023-06-26T14:00:00+01:00
+---
+
+We are excited to announce a new feature in the REANA web UI that takes user experience to a new level.
+Our interface now supports previewing of PDF and ROOT files directly from your browser.
+This means that instead of downloading these files to view them, you can simply click and inspect them right away,
+without leaving your browser!
+
+<!--more-->
+
+For a long time, the REANA web UI has allowed users to preview certain types of files present in the
+workflow workspace. This has proven to be very handy for users who want to quickly glance through
+images and text files in their workspaces without having to download them. We are now extending
+this capability to PDF and ROOT files as well.
+
+### PDF Files Preview
+
+Starting from REANA 0.9.1, when you navigate to a PDF file in your workflow workspace, an embedded 
+PDF viewer will load the document. You will be able to scroll through pages, zoom in and out,
+or even download the PDF if necessary. 
+You can use it to quickly inspect the content of any PDF generated by your analysis or uploaded to your
+workspace.
+Here is a sneak peek of how it looks:
+
+![PDF file preview](/images/pdf-files-preview.png)
+
+### ROOT Files Preview
+
+As for the ROOT files, we took an extra step forward to bring you a seamless preview experience.
+We have embedded a ROOT file viewer, based on [JSROOT](https://root.cern/js/), enabling you to inspect your
+ROOT files without leaving the browser window in which you browse your workspace.
+You can navigate through the file structure, interact with charts and ROOT objects, zoom on some details,
+and download the file if needed.
+Here is a small animation of the ROOT file viewer in action:
+
+![ROOT file preview](/images/root-files-preview.gif)
+
+These enhancements aim to save time and streamline the process of navigating and inspecting files
+in your REANA workflow workspace. We hope that they will prove beneficial in your research and data analysis.
+
+We are continually working on improving your experience with REANA, and we would love to hear your
+thoughts on these new features. If you have any feedback or suggestions, please don't hesitate to reach out.
+
+## See also:
+- [Previewing HTML file since REANA 0.8.1](/posts/2022/reana-0.8.1/)
```

Comment:
```suggestion
- [Previewing HTML files since REANA 0.8.1](/posts/2022/reana-0.8.1/#previewing-html-workspace-files-directly)
```
Let's link directly to the relevant section